pkg/review/commit_review_status.go

```diff
@@ -29,6 +30,9 @@ import (
 // when a Pull Request has been approved by a reviewer.
 const githubPRApproved = "APPROVED"
 
+// the default approval status we assign to a commit.
```

Comment:
Optional: you can combine these into a const block:

```
const (
  githubPRApproved = ...
  defaultApprovalStatus = ...
)

Reply:
Good suggestion, done.